ui/.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+    // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
```

Comment:
Should this German sentence about IntelliSense be in here?

Reply:
And what do you have against the German language? :-)